src/styles/main.scss

```
@@ -1,3 +1,69 @@
+@import './variable';
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: $general-font-size;
+}
+
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
   margin: 0;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  align-items: center;
+
+  flex-wrap: wrap;
+  gap: $gap;
+
+  width: $size-cell * 7 + $gap * 6;
+  margin: auto;
+  padding: 10px;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    box-sizing: border-box;
+    border: 1px solid black;
+
+    width: $size-cell;
+    height: $size-cell;
+    background-color: #eee;
+  }
+
+  &__day:hover {
+    cursor: pointer;
+    background-color: pink;
+
+    transform: translate(0, -20px);
+    transition-property: transform;
+    transition-duration: .5s;
```

move it to the `__day` bc you don't have transition on unhover
```suggestion
    background-color: pink;

    transform: translate(0, -20px);
```